src/components/Media/Media.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./Media.css";
+
+export default () => (
+  <div className="media">
```

Comment:
You are using the `className` as `media` here but class isn't being used to style any particular element and the CSS file is targeting `about`class instead of media if that's what you were trying to achieve. Please make the changes needed accordingly.